cypress/tests/core/basic/contact-form.js

```diff
@@ -0,0 +1,16 @@
+describe('Contact Form Tests', () => {
+  beforeEach(() => {
+    cy.autologin();
```

Comment:
Not needed to be logged in. Use common sense for it (or include one for logged in, another for anonymous).